pyproject.toml

```diff
@@ -1,14 +1,14 @@
 [tool.poetry]
 name = "saiph"
 # Also modify in saiph/__init__.py
-version = "1.5.4"
+version = "1.5.5"
 description = "A projection package"
 authors = ["Octopize <help@octopize.io>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
-pandas = "^1.3"
+pandas = ">=1.5.3"
```

Comment:
Why not forcing `^2.0.0`?

Reply:
Because `saiph` is a dependency and it's a pain when dependencies have hard constraints on another package.

Also, this allows us to update to pandas at a later stage in the API, while still benefiting from new features in saiph.